terraform/pas_terraform/modules/multi-tenancy/main.tf

```diff
@@ -0,0 +1,68 @@
+locals {
```

Comment:
Can we remove this module if not used? Are we planning to do group & group association as part of this GS?

Reply:
@hardik-rafay I can add this. Would adding a group and group association with a project admin role be sufficient?

Reply:
I've added a group and group association